OpenEdXMobile/src/main/java/org/edx/mobile/view/adapters/CourseHomeAdapter.kt

```diff
@@ -113,14 +111,22 @@ class CourseHomeAdapter(
                         R.drawable.edx_neutral_white_t_fill_neutral_dark_border,
                         binding.root.context.theme
                     )
-                    binding.rvSubSection.setVisibility(!otherSectionVisible)
-                    otherSectionVisible = true
+                    if (!isAnySectionVisible) {
+                        binding.rvSubSection.setVisibility(true)
```

Comment:
We can create a private method to update the visibility of a section based on a flag, as this thing is repeating on multiple instances in this class

Reply:
I have improved it in another way.